src/main/java/FrameworksDrivers/LogInView.java

```diff
@@ -55,13 +58,6 @@ public LogInView(JPanel masterPanel){
         signUpB.createButton(newPanel, "Sign Up", 40, 40, 40, 40);
 
 
-        //create action listener on button click
-//       logInB.addActionListener(this);
-//        setTitle("Log In");
-
-//        signUpB.addActionListener(this);
-//        setTitle("Sign Up");
-
         newPanel.setBackground(Color.lightGray);
         this.masterPanel.add(newPanel, "");
```

Comment:
You can replace the empty string with any string that you want. It might be better if you make the string something useful so that it is obvious that the string refers to the newPanel (perhaps make the string "newPanel")